exchanges/okx/ws_requests_test.go

```diff
@@ -0,0 +1,258 @@
+package okx
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+)
+
+func TestWSPlaceOrder(t *testing.T) {
+	t.Parallel()
+
+	_, err := ok.WSPlaceOrder(t.Context(), nil)
```

Comment:
```Error connecting auth socket: Okx websocket timeout waiting for response with signature login-response```
I will assume this is temporary for now, but please check that these tests still function

Reply:
That shouldn't be happening, I am getting `Okx websocket timeout waiting for response with signature 1` because my key 💩 itself. I can't seem to replicate this issue just yet.

Reply:
So its due to a code not in the list, it just doesn't get handled:
```
{"event":"error","msg":"Your IP is dumb and  is not in linking trusted IP addresses.","code":"50110","connId":"889ef657"}
```